src/Extension.php

```diff
@@ -85,6 +87,20 @@ public function setup() {
 		\add_action( 'plugins_loaded', [ $this, 'plugins_loaded' ], 5 );
 
 		\add_action( 'rcp_after_membership_admin_update', [ $this, 'rcp_after_membership_admin_update' ] );
+
+		/*
+		 * Filter if billing can be updated for membership.
+		 *
+		 * @link https://gitlab.com/pronamic-plugins/restrict-content-pro/-/blob/3.4.4/includes/memberships/class-rcp-membership.php#L3231-3240
+		 */
+		\add_filter( 'rcp_membership_can_update_billing_card', [ $this, 'rcp_membership_can_update_billing' ], 10, 2 );
```

Comment:
This filter is intended to indicate whether a credit card can be updated, if I understand correctly this has now been used more widely. It provides a link to the "Update Billing Card" page with the `[rcp_update_card]` shortcode. This shortcode really shows fields specifically for a credit card:

- Name on Card
- Card Number
- Card CVC
- Card ZIP or Postal Code
- Expiration (MM/YYYY)

https://github.com/stellarwp/restrict-content/blob/386b37e8fe58854c3816fd5c3de451a896737d15/core/templates/card-update-form-fields.php#L40

Using (or abusing) this filter to also update SEPA direct debit mandates does not seem to me to be sufficiently future-proof?

Reply:
Removed hooking into `rcp_membership_can_update_billing_card` filter in https://github.com/pronamic/wp-pronamic-pay-restrict-content-pro/pull/23/commits/0c46dfe43fc27d2df6800f287e3141988d38fba7.